test/e2e/testdata/vault/vault.yaml

```
@@ -0,0 +1,103 @@
+---
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)